database/rrdhost.c

```diff
@@ -889,6 +889,21 @@ struct label *create_label(char *key, char *value, LABEL_SOURCE label_source)
     return result;
 }
 
+void clean_label_list(RRDHOST *host, struct label *new_labels)
```

Comment:
This isn't a cleaning operation -> a better name would be `replace_label_list`.

But, as I'm about to explain above, I don't understand why you've factored this out of the reload function to call from another file.

Reply:
When we are receiving the stream from another host, Netdata will update the `RRDHOST` for that connection instead the `localhost`, this will force us to remove the previous labels that were set when the connection started and we will load a new, this is the motive I began to export this function.